security-oauth2/src/main/java/io/micronaut/security/oauth2/endpoint/token/response/IdTokenLoginHandler.java

```diff
@@ -32,17 +32,15 @@
 import io.micronaut.security.errors.PriorToLoginPersistence;
 import io.micronaut.security.token.cookie.AccessTokenCookieConfiguration;
 import io.micronaut.security.token.cookie.CookieLoginHandler;
+import io.micronaut.security.token.cookie.LoginCookieProvider;
+import jakarta.inject.Inject;
 import jakarta.inject.Singleton;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.text.ParseException;
 import java.time.Duration;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
```

Comment:
We need a checkstyle rule for this so that we don't always have diffs changing this back and forth because of different IDE settings.

(I vote for banning star imports)

Reply:
resolved via https://github.com/micronaut-projects/micronaut-security/pull/1826/commits/629b8fee4acdef3db4ac2e8694b4d74d83825a9d